noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -8,8 +8,10 @@ use dep::aztec::{
     },
     oracle::random::random,
     protocol_types::{
-        address::AztecAddress, constants::GENERATOR_INDEX__NOTE_NULLIFIER,
-        hash::poseidon2_hash_with_separator, traits::Serialize,
+        address::AztecAddress,
+        constants::GENERATOR_INDEX__NOTE_NULLIFIER,
+        hash::poseidon2_hash_with_separator,
+        traits::{Packable, Serialize},
```

Comment:
All the files with notes now generally need to import `Packable` as because of the change in `noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr`.

See comment in that file for details.

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/11602